lib/PuppeteerSharp/PuppeteerSharp.csproj

```diff
@@ -39,11 +39,7 @@
     <CodeAnalysisRuleSet>../PuppeteerSharp.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
   <PropertyGroup>
-    <DocumentationFile>bin\$(Configuration)\netstandard2.0\PuppeteerSharp.xml</DocumentationFile>
```

Comment:
no need for full path here. GenerateDocumentationFile sorts that for you. and will also work if u ever decide to multitarget frameworks